src/injections/css/bug1710701-tiktok.com-comments-scrollbar.css

```diff
@@ -0,0 +1,8 @@
+/**
+ * www.tiktok.com - Ensure that the comment section for videos has a vertical
+ *                  scrollbar when needed.
+ */
```

Comment:
Could you please add the link to webcompat and bugzilla for the reference?

Reply:
Yes, odd that I forgot it :)